examples/04_glm_first_level/plot_adhd_dmn.py

```diff
@@ -18,8 +17,8 @@
 from nilearn import datasets, plotting
 from nilearn.input_data import NiftiSpheresMasker
 
-from nilearn.stats.first_level_model import FirstLevelModel
-from nilearn.stats.first_level_model import make_first_level_design_matrix
+from nilearn.glm.first_level import FirstLevelModel
```

Comment:
Should we also rename `FirstLevelModel` to `FirstLevel` to match ?

Reply:
I feel like it loses clarity to just call it a `FirstLevel`. If we take [sklearn.linear_model](https://scikit-learn.org/stable/modules/classes.html#module-sklearn.linear_model) as an example, most of them have a second word like "Regression" unless the first part of the name is really iconic like "Lasso". So I'd stick with `FirstLevelModel`.

`FirstLevelGLM` would also be clear.